map/place_page_info.cpp

```diff
@@ -163,4 +163,41 @@ string Info::GetApproximatePricing() const
 }
 
 void Info::SetMercator(m2::PointD const & mercator) { m_mercator = mercator; }
+
+Banner const & Info::GetBanner() const
+{
+  return m_banner;
+}
+
+Banner::Banner(feature::Metadata const & metadata) : m_metadata(metadata) {}
+
+bool Banner::IsActive() const
+{
+  auto const currentTime = time(nullptr);
+  auto const startTime = currentTime - 1;
+  auto const endTime = currentTime + 1;
+
+  return startTime < currentTime && endTime > currentTime;
+}
+
+string Banner::TitleId() const
+{
+  return "lose_your_money";
+}
+
+string Banner::MessageId() const
+{
+  return "casino_invite_you";
+}
+
+string Banner::IconId() const
+{
+  return "sweet_babe";
+}
+
+string Banner::Url() const
+{
+  return "http://some_url";
+}
+
```

Comment:
It's much better to write a code that parses metadata here, and returns loaded strings. Feel free to comment parts that use particular metadata keys, as they're not in the repository now.